cmd/operator_install.go

```diff
@@ -36,7 +36,9 @@ func newOperatorInstallCmd(operatorConfig *operatorCommandConfig) *cobra.Command
 		Short: "Install the Appsody Operator.",
 		Long: `Install the Appsody Operator into your configured Kubernetes cluster.
 		
-The Appsody Operator listens for incoming AppsodyApplication resources on your cluster. For more information, see https://operatorhub.io/operator/appsody-operator. By default, the operator only watches one namespace, unless the '--watch-all' flag is used. Additional operators must be installed to watch additional namespaces.`,
+The Appsody Operator listens for incoming AppsodyApplication resources on your cluster. For more information, see https://operatorhub.io/operator/appsody-operator. 
+
+By default, the operator only watches one namespace. You can specify the '--watch-all' flag is used to tell the Appsody Operator to watch all namspaces. If you want to match multiple namespaces, but not all namespaces, additional operators must be installed to watch additional namespaces.`,
```

Comment:
The third sentence isn't quite clear either and I think it says match where it means watch...
"If you want to match multiple namespaces, but not all namespaces, additional operators must be installed to watch additional namespaces."?

Can we change the whole piece to read as follows...?

By default, the operator watches a single namespace. You can specify the ‘--watch-all’ flag to tell the operator to watch all namespaces in the cluster. If you want to watch multiple, but not all, namespaces within your cluster, install an additional operator to watch each additional namespace."